evalkit_api_client/evalkit_api_client/v1_client.py

```diff
@@ -0,0 +1,172 @@
+import logging
+import requests
+from requests import Response
+from typing import Any, Dict, Iterator, Optional
+from urllib.parse import urljoin
+
+from evalkit_api_client.errors import APIPaginationException
+from evalkit_api_client.interface import EvalKitAPIClient
+from evalkit_api_client.types import (
+        PaginatedResponse, RequestHeaders, RequestParams)
+
+logger = logging.getLogger()
+
+
+class EvalKitAPIv1(EvalKitAPIClient):
+    """
+    Evalkit v1 API Client.
+
+    This client can be used to make requests to the v1 Evalkit API.
+
+    Create separate clients for other versions of the Evalkit API.
+    """
+
+    def __init__(self,
+                 api_url: str,
+                 api_token: Optional[str] = None) -> None:
+        """
+        Creates a evalkit API client given a base URL for the API, an optional
+        API token, and an optional requests library.
+
+        The optional requests library should be either the python HTTP requests
+        library or the equivalent (e.g. a Requests-OAuthlib session object).
+        """
```

Comment:
Update the documentation to clarify that you are not actually accepting an optional requests library

Reply:
handled internally :-)